backend/models/displaySettings.ts

```
@@ -0,0 +1,40 @@
+import { model, Schema } from 'mongoose'
+import { DisplaySettings, Locale, locales } from '../../common/types.js'
+
+const overwrites = {} as { [key in Locale]: { type: typeof Schema.Types.Mixed; required: true; default: Function } }
```

Biome

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)